geoapps/inversion/params.py

```diff
@@ -135,6 +135,11 @@ def __init__(
 
         super().__init__(input_file=input_file, **kwargs)
 
+        if not self.forward_only:
+            for key in self.__dict__:
+                if "channel_bool" in key and getattr(self, key[:-5], None) is not None:
```

Comment:
do the apps still need the channel_bools for inversion?  I thought we took that out?